### ISSUE TITLE
Deletion Confirmation Keeps Recurring After I've Clicked No

### ISSUE THREAD

**Issue by awasikowski** [phw198/OutlookGoogleCalendarSync#622]
:memo: Use the below template to report your issue.  
:bulb: Upgrade to the latest alpha release to check if the problem is already fixed.  
:paperclip: To aid investigation, provide your OGcalsync.log file ([see instructions](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Reporting-problems#locating-your-ogcalsynclog-file)).  

----

**OGCS Version**: v1.2.3.4  
**Installed or Portable**: Installed

**Problem Description**  
\When the Deletion Confirmation message appears for the first time during a sync asking if I want to delete a calendar event, if I click No, the Deletion Confirmation message will appear again for the same event(s) during the next sync.  If I continue to click No, it will continue to appear for the same event(s) each time a sync occurs.

This is happening with events that have any category, even categories I have selected in the Settings to exclude from my sync, or no category at all.  I do not want to delete these events as they are legitimate events
![ogcs deletion confirmation msg](https://user-images.githubusercontent.com/44587794/47678592-ceb77e80-db87-11e8-8c06-3cd51c890ca0.png).

[OGcalsync.log](https://github.com/phw198/OutlookGoogleCalendarSync/files/2526979/OGcalsync.log)

**Steps to Reproduce the Issue**  
1. \Reproduces automatically with each sync
1. 

**Any other information**  
\I would like to suggest an update that fixes this.  When I click No to deleting an event, I don't want to keep getting asked.

**Comment by phw198** [comment]
Sounds like you need to use the "merge" option under `Settings > How`?

**Comment by phw198** [comment]
Right OK, in which case the only situation OGCS should want to delete an item with 2-way sync configured is if the item was already synced and then for later syncs that item is no longer in scope.

For example I could see this order of events causing such an issue:
1. 2-way sync everything, syncing item x with category y to Google
1. Now set category y as not included for sync
1. Google item x will be prompted for deletion.

Also if you manually deleted a synced item in Google because you did not want it in Google, that will be synced back to Outlook too with a delete prompt for Outlook. Might that have happened?

For either of these situations, the only way to recover would be to manually recreate the calendar item afresh and let OGCS delete the original.

**Comment by awasikowski** [comment]
Unfortunately, no, none of those situations has happened.

I am currently getting the Deletion Confirmation messages on one event shown below.
![image](https://user-images.githubusercontent.com/44587794/47745161-a3e12f00-dc48-11e8-9068-72cf48550c7a.png)

Although this event is now in the past, I've been getting it all day with each sync as I just got it now.  This particular event had the category Shared Appointments at the time it was created and that category has always been selected in Settings to be included.

Here is a screen shot of my settings, which show Shared Appointments being included:
![image](https://user-images.githubusercontent.com/44587794/47745487-63ce7c00-dc49-11e8-8e88-344462426b61.png)

Here is a screen shot of the Settings > Sync Options showing I do not want to sync any events in the past, although I'm not sure if this particular setting is only looking at whole days vs hours in the past.
![image](https://user-images.githubusercontent.com/44587794/47745980-975dd600-dc4a-11e8-9cc8-93f46f2998a4.png)

Currently, most events are being created in Outlook, then syncing to Google.  When I initially installed OGCS and went through the first sync, I did end up deleting duplicate entries, but I have not deleted any entries since.

**Comment by kwagner115** [comment]
I too have seen the same message on a few occasions for no apparent reason. I am not sure I can replicate.

**Comment by phw198** [comment]
> I did end up deleting duplicate entries  

How do you know you deleted the "right" duplicate? Only one of the two will be linked to the synced item in the source calendar (Outlook). If you manually deleted the "wrong" one it would then want to sync that deletion back to Outlook. If there are duplicates, nearly always best to let OGCS determine which one to delete.

Regardless, [the wiki has guidance](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/Hints-&-Tips#fixing-a-stubborn-calendar-sync-with-full-sync) on how to resolve an item that constantly prompts for deletion approval.

**Comment by fogoat** [comment]
I accidentally set Outlook to Google one-way by mistake. I actually meant to do it the other way around. I deleted the meeting in Google Calendar. I set the one-way to go Google to Outlook now. I tried Sync and Full Sync (shift button press + Sync) and both are asking me to confirm deletion. I don't want to delete and re-create since I did not create the meeting. I ended up duplicating the meetings in Outlook and then allow ogcs to delete the meetings.
